lib/stytch/b2b_discovery.rb

```diff
@@ -123,8 +123,8 @@ def exchange(
         locale: nil
       )
         request = {
-          intermediate_session_token: intermediate_session_token,
-          organization_id: organization_id
+          intermediate_session_token:,
```

Comment:
My local version of rubocop prefers this syntax. I tested it out in irb and it looks valid, but I'm not sure why this change is happening now?

```
maxwellgerber@MacBook-Pro-103 stytch-ruby % irb
irb(main):001:0> foo = 1
=> 1
irb(main):002:0> test = { foo: }
=> {:foo=>1}
````